valgrind.supp

```diff
@@ -0,0 +1,519 @@
+   fun:add_getset
+   fun:PyType_Ready
+   fun:type_new
+   fun:type_call
+   fun:PyObject_Call
+   fun:call_function_tail
+   fun:PyObject_CallFunction
+}
+{
+   From NumPy 1.3.0 (python 2.6)
+   Memcheck:Leak
+   fun:malloc
+   fun:PyString_FromString
+   fun:PyString_InternFromString
+   fun:descr_new
+   fun:PyDescr_NewGetSet
+   fun:add_getset
+   fun:PyType_Ready
+   fun:PyObject_GenericGetAttr
+   fun:PyObject_GetAttr
+   fun:PyObject_GetAttrString
+   fun:PyObject_CallMethod
+   fun:Py_InitializeEx
+}
+{
+   From NumPy 1.3.0 (python 2.6)
+   Memcheck:Leak
+   fun:malloc
+   fun:PyString_FromString
+   fun:PyString_InternFromString
+   fun:descr_new
+   fun:PyDescr_NewMethod
+   fun:add_methods
+   fun:PyType_Ready
+   fun:PyObject_GenericGetAttr
+   fun:PyObject_GetAttr
+   fun:PyObject_GetAttrString
+   fun:PyObject_CallMethod
+   fun:Py_InitializeEx
+}
+{
+   From NumPy 1.3.0 (python 2.6)
+   Memcheck:Leak
+   fun:malloc
+   fun:PyString_FromString
+   fun:PyType_Ready
+   fun:PyObject_GenericGetAttr
+   fun:PyObject_GetAttr
+   fun:builtin_getattr
+   fun:PyCFunction_Call
+   fun:call_function
+   fun:PyEval_EvalFrameEx
+   fun:gen_send_ex
+   fun:gen_iternext
+   fun:PyIter_Next
+}
+{
+   From NumPy 1.3.0 (python 2.6)
+   Memcheck:Leak
+   fun:malloc
+   fun:PyString_FromString
+   fun:method_get_doc
+   fun:getset_get
+   fun:PyObject_GenericGetAttr
+   fun:PyObject_GetAttr
+   fun:PyEval_EvalFrameEx
+   fun:fast_function
+   fun:call_function
+   fun:PyEval_EvalFrameEx
+   fun:PyEval_EvalCodeEx
+   fun:function_call
+}
+{
+   From DistNumPy (python 2.6) - non-freed in slave processes.
+   Memcheck:Leak
+   fun:realloc
+   fun:list_resize
+   fun:listextend
+   fun:call_function
+   fun:PyEval_EvalFrameEx
+   fun:PyEval_EvalCodeEx
+   fun:PyEval_EvalCode
+   fun:PyImport_ExecCodeModuleEx
+   fun:load_source_module
+   fun:load_module
+   fun:load_package
+   fun:load_module
+}
+{
+   From DistNumPy (python 2.6) - non-freed in slave processes.
+   Memcheck:Leak
+   fun:malloc
+   fun:PyList_New
+   fun:do_mklist
+   fun:do_mkvalue
+   fun:va_build_value
+   fun:Py_BuildValue
+   fun:load_package
+   fun:load_module
+   fun:import_submodule
+   fun:load_next
+   fun:import_module_level
+   fun:PyImport_ImportModuleLevel
+}
+{
+   From DistNumPy (python 2.6) - non-freed in slave processes.
+   Memcheck:Leak
+   fun:malloc
+   fun:fill_free_list
+   fun:PyInt_FromLong
+   fun:long_int
+   fun:PyNumber_Int
+   fun:int_new
+   fun:type_call
+   fun:PyObject_Call
+   fun:do_call
+   fun:call_function
+   fun:PyEval_EvalFrameEx
+   fun:fast_function
+}
+{
+   From DistNumPy (python 2.6) - non-freed in slave processes.
+   Memcheck:Leak
+   fun:malloc
+   fun:fill_free_list
+   fun:PyInt_FromLong
+   fun:PyLong_FromVoidPtr
+   fun:init_ctypes
+   fun:_PyImport_LoadDynamicModule
+   fun:load_module
+   fun:import_submodule
+   fun:load_next
+   fun:import_module_level
+   fun:PyImport_ImportModuleLevel
+   fun:builtin___import__
+}
+
```

Comment:
In the val_clean.txt you attached at the end of file I see this:
==115610== 17 bytes in 1 blocks are definitely lost in loss record 44 of 1,532
==115610==    at 0x4C29BE3: malloc (vg_replace_malloc.c:299)
==115610==    by 0x67AC949: strdup (strdup.c:42)
==115610==    by 0x472529: main (pbsd_main.c:1984)

Do we need this as well in suppression file?